src/Loading/Plugins/babylon.babylonFileLoader.ts

```diff
@@ -529,6 +529,16 @@
                     }
                 }
 
+                // Environment texture
+                if (parsedData.environmentTexture !== undefined && parsedData.environmentTexture !== null) {
+                    scene.environmentTexture = Texture.Parse(parsedData.environmentTexture, scene, rootUrl);

```

Comment:
Don't we want to call CreateFromFilteredData here?

Reply:
When using CreateFromFilteredData in Sandbox, need to add "file:" before texture name.